zuul.d/jobs.yaml

```diff
@@ -0,0 +1,38 @@
+---
+- job:
+    name: infra-operator-content-provider
+    parent: content-provider-base
+    vars:
+      cifmw_operator_build_org: openstack-k8s-operators
+      cifmw_operator_build_operators:
+        - name: "openstack-operator"
+          src: "~/src/github.com/{{ cifmw_operator_build_org }}/openstack-operator"
+          image_base: infra
+    irrelevant-files: &irrev
+      - tests/kuttl
+      - docs
+      - containers/ci
+      - .github/workflows
+      - .ci-operator.yaml
+      - .dockerignore
+      - .gitignore
+      - .golangci.yaml
+      - .pre-commit-config.yaml
+      - LICENSE
+      - OWNERS*
+      - PROJECT
+      - .*/*.md
+      - kuttl-test.yaml
+      - renovate.json
+
+- job:
+    name: infra-operator-crc-podified-edpm-deployment
+    parent: cifmw-crc-podified-edpm-deployment
+    vars:
+      bmo_setup: true
```

Comment:
It is no longer needed, we can drop it.

Reply:
Why was that var added in the first place to crc jobs? It's by default true.

Reply:
this var was used with the older [crc parent](https://github.com/rdo-infra/review.rdoproject.org-config/blob/master/zuul.d/_jobs-crc.yaml#L4) and consumed [here](https://github.com/rdo-infra/review.rdoproject.org-config/blob/master/playbooks/crc/openstack.yaml#L39). We will be cleaning up soon.